store/actions/profiles.js

```diff
@@ -58,13 +60,18 @@ export function createProfile(
   image
 ) {
   return async (dispatch, getState) => {
-    const token = getState().auth.token;
-    const userId = getState().auth.userId;
-
+    // const token = getState().auth.token;
```

Comment:
We don't need auth tokens anymore?

Reply:
It's automatically handled by the SDK, that's why I've also refactored sign-in/sign-out.